### ISSUE TITLE
When I can finish the training/fine-tuning process?

### ISSUE THREAD

**Issue by Toland12** [jingyuanli001/RFR-Inpainting#2]
Hi, thanks for your shared work.
I try to train a new model with some additional masks in Celeba-HQ, and I found there is no STOP ITERS NUMBER for training/fine-tuning process. Could you plz share when I can finish the training/fine-tuning?

**Comment by jingyuanli001** [comment]
> Hi, thanks for your shared work.
> I try to train a new model with some additional masks in Celeba-HQ, and I found there is no STOP ITERS NUMBER for training/fine-tuning process. Could you plz share when I can finish the training/fine-tuning?

Thanks for pointing this out, we will update the code to include this. For Celeb-HQ data, you can start finetuning after 350,000 iterations and end training after 600,000 iterations. But this suggestion is only for the case where the batch size is 6 (i.e. 3,600,000 images have passed though the network during training). More iterations are needed for smaller batch size.
